src/datasources/data-frame/README.md

```diff
@@ -1,6 +1,11 @@
 # Systemlink Data Frame data source
 
-<!--
-    TODO: Write a short description of your plugin and document any extra
-    configuration that is required for development.
--->
+This plugin will have three main workflows:
+
+- Populating a
+  [query variable](https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#add-a-query-variable)
+  with a list of user-accessible systems
+
+- Visualizing system connection status counts on a dashboard
+
+- Visualizing system metadata on a dashboard
```

Comment:
This is the wrong plugin's README.

Also, let's use the present tense here. Maybe something like:

```suggestion
This is a plugin for the Systems Management service. It allows you to:

- Populate a
  [query variable](https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#add-a-query-variable)
  with a list of systems connected to the server
- Visualize system connection status counts on a dashboard
- Visualize system metadata on a dashboard
```